pkg/kind/kind.go

```diff
@@ -110,6 +115,46 @@ func checkDocker() error {
 	return nil
 }
 
+func createLocalRegistry() error {
+	deleteContainerRegistry := deleteContainerRegistry()
+	if err := deleteContainerRegistry.Run(); err != nil {
+		return fmt.Errorf("failed to delete local registry: %w", err)
+	}
+	localRegCheck := exec.Command(
+		"docker", "run", "-d", "--restart=always", "-p", "127.0.0.1:"+container_reg_port+":5000",
+		"--name", container_reg_name, "registry:2",
+	)
+	if err := localRegCheck.Run(); err != nil {
+		return fmt.Errorf("failed to create local registry container: %s", err.Error())
```

Comment:
```suggestion
		return fmt.Errorf("failed to create local registry container: %w", err.Error())
```

to get the lint action passing